interfaces/cython/cantera/thermo.pyx

```diff
@@ -290,6 +285,46 @@ cdef class ThermoPhase(_SolutionBase):
     def __call__(self, *args, **kwargs):
         print(self.report(*args, **kwargs))
 
+    property _is_pure:
+        """
+        The phase represents a pure (fixed composition) substance
+        """
+        def __get__(self):
+            return self.thermo.isPure()
+
+    property _is_compressible:
+        """
+        The density of the phase is an independent variable
+        """
+        def __get__(self):
+            return self.thermo.isCompressible()
```

Comment:
These methods are unused. Are they needed?

Reply:
`_is_pure` is used in unit tests. `_is_compressible` is indeed not used, but I probably should add a unit test using this also.

Reply:
I am making them public as I view these 'service' functions as essential for testing purposes (i.e. debugging or unit tests).

Reply:
I am making them public as I view these 'service' functions as essential for testing purposes (i.e. debugging or unit tests).

Reply:
PS: while I made those methods public, a similar rationale applies to `_native_state`, `_full_state` and `_partial_states`. I'd still keep them 'hidden' though.